### PR TITLE
StyleBox: Document correct methods to draw with a RID

### PR DESCRIPTION
**Pull request by jmpelletier**
Object.get_instance_id returns an int, and it's not clear how this can be used to obtain an RID. The get_canvas_item method is the simplest way of obtaining a valid RID in this context.

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]

File: doc/classes/StyleBox.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
 			<argument index="1" name="rect" type="Rect2" />
 			<description>
 				Draws this stylebox using a [CanvasItem] with given [RID].
```
Comment (akien-mga) [inline doc/classes/StyleBox.xml:49]:
```suggestion
				Draws this stylebox using a [CanvasItem] with the given [RID].
```

Comment (akien-mga) [inline doc/classes/StyleBox.xml:49]:
```suggestion
				Draws this stylebox using a [CanvasItem] identified by the given [RID].
```

File: doc/classes/StyleBox.xml [hunk 2]
```diff
@@ -47,7 +47,7 @@
...
-				You can get a [RID] value using [method Object.get_instance_id] on a [CanvasItem]-derived node.
+				You can get a [RID] value using the [method CanvasItem.get_canvas_item] method of [CanvasItem]-derived nodes.
```
Comment (akien-mga) [inline doc/classes/StyleBox.xml:50]:
```suggestion
				The [RID] value can either be the result of [method CanvasItem.get_canvas_item] called on an existing [CanvasItem]-derived node, or directly from creating a canvas item in the [RenderingServer] with [method RenderingServer.canvas_item_create].
```

**Review by akien-mga — COMMENTED** [review]
Actually this can be improved further, see my new suggestions.

**Comment by akien-mga** [comment]
Did you mean to close this?

**Comment by jmpelletier** [comment]
> 

Oops, pulled the trigger a bit too quickly on this. :(

**Comment by akien-mga** [comment]
I force pushed an update to squash the commits and tweak the commit message for clarity.

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**